src/static-site/about.html

```diff
@@ -1,42 +1,16 @@
-<!DOCTYPE html>
-
-<html lang="en">
-<head>
-    <meta charset="utf-8">
-    <meta content="IE=edge" http-equiv="X-UA-Compatible">
-    <meta content="width=device-width, initial-scale=1" name="viewport">
-    <meta content="Let's elect a Brand New Congress that works for all." name="description">
-    <meta content="brand, new, congress, bernie, sanders, political, revolution, elect, progressive, 2016, democrat" name="keywords">
-    <meta content="Brand New Congress" name="author">
-    <!--     Meta tags for Facebook   -->
-    <meta content="http://brandnewcongress.org" property="og:url">
-    <meta content="website" property="og:type">
-    <meta content="Brand New Congress" property="og:title">
-    <meta content="Let's elect a Brand New Congress that works for all." property="og:description">
-    <meta content="http://brandnewcongress.github.io/assets/img/bnc-fb-share-img.png" property="og:image">
-
-    <!-- font-awesome-icon-font-->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
-
-    <title>Brand New Congress</title>
-    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700,800,300,600,400italic' rel='stylesheet' type='text/css'>
-    <link href="/static-assets/css/toolkit-minimal.css" rel="stylesheet">
-    <link href="/static-assets/css/application-minimal.css" rel="stylesheet">
-</head>
-
-<body>
-
-
-    <!-- Block 1 -->
+    <div class="container">
+      <div class="quote1">“This campaign is about more than electing a president,</div>
+      <div class="quote2">It is about a political revolution.”</div>
+      <div class="quote3">Bernie Sanders (I-VT)</div>
+    </div>
 
 
-    <div class="block m-y-0 p-y-0 p-x-0 p-t-md text-center">
-        <div class="container-fluid secondary-img"><a href="/"><img alt="Let's elect a Brand New Congress that works for all." src="/static-assets/img/hdr-circles-map-988x316@2x.gif"></a>
-        </div>
-
+    <div class="container">
```

Comment:
This div should be deleted (it is repeated twice) - I do think it is a nice quote